BSP/Simulator/DataGeneration/simulate.py

```diff
@@ -200,7 +200,7 @@ def main():
         except KeyboardInterrupt:
             curses.endwin()
             if BeVolt is not None:
-                print("\n\rWould you like to change 'wires', 'quit', or 'PLL'?")
+                print("\n\rWould you like to change 'wires', 'quit', 'PLL', or send a CAN message ('CAN')?")
```

Comment:
This choice menu is getting a little long. Probably fine for now, but if much more is to be added, I suggest doing a menu sort of thing.
```
Select a #: 
1. Wires
2. PLL
3. ...
```

Reply:
Good idea. I think I might as well implement this now.